mlrun/common/model_monitoring/helpers.py

```
@@ -53,22 +51,6 @@ def parse_monitoring_stream_path(
         else:
             stream_uri += f"?topic=monitoring_stream_{project}_{function_name}"
 
-    elif stream_uri.startswith("v3io://") and mlrun.mlconf.is_ce_mode():
```

I agree that the http is not relevant any more... but lets raise an exception if the stream is not v3io or kafka

---

When this function is called the `stream_uri` is already set, so I think validating it in the first steps will give better coverage https://github.com/mlrun/mlrun/pull/6983#discussion_r1897325932